src/market/service.ts

```diff
@@ -28,6 +27,12 @@ export class MarketService {
   private logger?: Logger;
   private taskPackage?: Package;
   private maxResubscribeRetries = 5;
+  private proposalsCount = {
+    initial: 0,
+    filtered: 0,
```

Comment:
Filtered means accepted or rejected? The naming is confusing.

Reply:
filtered is not used, it was left by mistake